invenio_search_ui/assets/semantic-ui/js/invenio_search_ui/components/SearchDropdowns.js

```diff
@@ -62,7 +62,7 @@ export class DropdownSort extends Component {
         trigger={
           <span className="flex align-items-center">
             {this.getCurrentlySelectedOption(parsedOptions).text}
-            <Icon name="dropdown" />
+            <Icon name="dropdown" role="option" />
```

Comment:
Sorry, late to the party, but shouldn't the `role="option"` be on the `<span>` in this case? (The `Icon` is hidden from screen readers, and doesn't give any information to the user).